LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageDispatcher.cs

```diff
@@ -50,17 +47,21 @@ public sealed class MessageDispatcher : IDisposable, IMessageDispatcher
         }
 
         /// <summary>
-        /// Use this constructor only for tests.
+        /// Use this only for tests.
```

Comment:
If this is designed to be used from tests only then doesn't it belong in `Tests/Common/LoRaWan.Tests.Common.csproj` instead?

Reply:
Moved it to its own type in Tests.Common with 9ea05dc05bad78a3a9df60fd16d4dc0990559098